cds_migrator_kit/videos/weblecture_migration/transform/xml_processing/quality/dates.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# CDS-Videos is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-Videos dates migration module."""
+
+
+from datetime import datetime
+
+
+def parse_date(date_str):
```

Comment:
i discovered recently edtf parsers, maybe you want to check it as well, or arrow module. They are insanely powerful in disambiguating human readable date formats, it would save you time from trying to validate against "known" valid formats.